Apps/JobScheduler/src/Jobs/CloseAccountJob.cs

```diff
@@ -55,6 +57,9 @@ public class CloseAccountJob
         private readonly int hoursBeforeDeletion;
         private readonly string emailTemplate;
         private readonly string host;
+        private const string authConfigSectionName = "ClientAuthentication";
```

Comment:
This doesn't follow constant variable naming standards

Reply:
it does now!